app/views/staff/transactions/_form.html.haml

```diff
@@ -0,0 +1,24 @@
+= f.govuk_error_summary
+= f.govuk_text_field :reference
+= f.govuk_text_area :description
+= f.govuk_collection_select :transaction_type,
+                               yaml_to_objects("transaction", "transaction_type"),
+                               :code,
+                               :name,
+                               label: { text: t("form.transaction.transaction_type.label") },
```

Comment:
Leeky left me some feedback today on how we can move these labels to the active record locale and get additional benefits:

- labels don't need to be passed in on each form
- error messages include consistent field names
- form locale variables are grouped up (our en.yml is getting busy)

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/77#discussion_r355410041

I ended up doing https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/config/locales/models_and_forms.en.yml#L6 which doesn't include hints as it's not nicely supported yet.